### PR TITLE
Improve examples in README.md

### PR DESCRIPTION
**Pull request by tomaszkane**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌

### CODE REVIEW

**Review by vjik — APPROVED** [review]

**Review by xepozz — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -41,16 +41,18 @@ If not, then in the simplest use case in your console entry script do the follow
...
 
 require_once __DIR__ . '/vendor/autoload.php';
 
-$app = new Yiisoft\Yii\Console\Application();
+$app = new Application();
 
 $app->setCommandLoader(new CommandLoader(
-    // Any container implementing `Psr\Container\ContainerInterface`.
-    new PsrContainer([MyCustomCommand::class => new MyCustomCommand()]),
+    // Any container implementing `Psr\Container\ContainerInterface` in example:
```
Comment (devanych) [inline README.md:54]: // Any container implementing `Psr\Container\ContainerInterface` for example:

File: README.md [hunk 2]
```diff
@@ -41,16 +41,18 @@ If not, then in the simplest use case in your console entry script do the follow
...
+    new Container(ContainerConfig::create()),
     // An array with command names as keys and service ids as values.
```
Comment (devanych) [inline README.md:56]: // An array with command names as keys and service ids as values:

**Review by devanych — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -41,17 +41,19 @@ If not, then in the simplest use case in your console entry script do the follow
...
-$app = new Yiisoft\Yii\Console\Application();
+$app = new Application();
 
 $app->setCommandLoader(new CommandLoader(
-    // Any container implementing `Psr\Container\ContainerInterface`.
-    new PsrContainer([MyCustomCommand::class => new MyCustomCommand()]),
-    // An array with command names as keys and service ids as values.
+    // Any container implementing `Psr\Container\ContainerInterface` for example:
+    new Container(ContainerConfig::create()),
+    // An array with command names as keys and service ids as values:
```
Comment (vjik) [inline README.md:56]:
```suggestion
    // An array with command names as keys and service IDs as values:
```

**Comment by vjik** [comment]
Thank you 👍

**Pull request merged**